lib/schemaUtils.js

```diff
@@ -2143,6 +2213,12 @@ module.exports = {
     return { url: reqUrl, pathVars, collectionVars };
   },
 
+  cleanWebhookName: function(webhookName) {
```

Comment:
is this just a webhook related function or could it be treated more generic like cleanName?

Reply:
Just related to Webhooks